.github/workflows/manual-sync.yml

```diff
@@ -51,8 +51,8 @@ jobs:
          # https://stackoverflow.com/questions/14700579/bash-converting-string-to-boolean-variable
          function boolean () {
             case $1 in
```

Comment:
how about this instead?
```sh
case `echo $1 | tr '[:upper:]' '[:lower:]'` in
```

Reply:
I was leaning on K&R but I'll see if that fits better

Reply:
I'll keep this as it is for now since I'm using Bash-native patterns here which should reduce the point of failure.